src/DB_Command.php

```diff
@@ -285,6 +285,18 @@ public function query( $args, $assoc_args ) {
 	 *     # Export all tables matching prefix
```

Comment:
Can we name this argument `--exclude_tables=<tables>` instead of `--skip-tables=<tables>`?

The reason I'm leaning towards the former is that we already have the paradigm of `--skip-plugins=<plugins>` and `--skip-themes=<themes>`, which are framework flags and functionally different than what this feature will do.

Related https://github.com/wp-cli/extension-command/issues/14

Also, this argument should be present in `## OPTIONS` above.